test/e2e/manifests/meteringconfigs/nfs.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: metering.openshift.io/v1
+kind: MeteringConfig
+metadata:
+  name: operator-metering
+spec:
+  logHelmTemplate: true
+
+  storage:
+    type: hive
+    hive:
+      type: sharedPVC
+      sharedPVC:
+        createPVC: true
+        storageClass: nfs-server
+        size: 5Gi
+
+  reporting-operator:
```

Comment:
Couldn't you pare this down by just going with all defaults below here?

Reply:
I would agree for this most part, except for the metrics importer configuration, but even that attempts to set a few default values, and we do this throughout most of the MeteringConfig YAML manifests anyways, and fixing that is out of the scope of this PR anyways.